service/integration/open_pbs/geopm_install_pbs_power_limit.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#  Copyright (c) 2015 - 2023, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
```

Comment:
Allow passing a "--remove" option to uninstall the hook, remove the "save file" directory.